### PR TITLE
chore: improve section performance

### PR DESCRIPTION
**Pull request by nathggns**
Improve section performance by:

- Preventing multiple simultaneous saved article syncing (triggering multiple re-renders)
- Improve performance of image placeholder component
- Improve performance of articule summary component

### CODE REVIEW

File: packages/image/src/logo/t.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import React from "react";
 import { Image } from "react-native";
 
+// eslint-disable-next-line global-require
+const logoPath = require("../../assets/t.png");
```
Comment (tuncaulubilge) [inline packages/image/src/logo/t.js]: Can this be just a regular import? `import logoPath from "../../assets/t.png";`

File: packages/related-articles/src/related-article-item.base.js [hunk 2]
```diff
@@ -141,13 +141,13 @@ class RelatedArticleItem extends Component {
                 )
               }
               datePublicationProps={{ date: publishedTime, showDay: false }}
-              headline={() => (
```
Comment (parkjh) [inline packages/related-articles/src/related-article-item.base.js]: can we have an eslint rule to prevent this?

Comment (nathggns) [reply packages/related-articles/src/related-article-item.base.js]: Not as part of this PR – its occasionally a valid pattern too

File: packages/image/src/logo/t.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 import { Image } from "react-native";
+import logoPath from "../../assets/t.png";
 
 function T() {
   return (
     <Image
       fadeDuration={0}
       resizeMode="contain"
-      // eslint-disable-next-line global-require
-      source={require("../../assets/t.png")}
```
Comment (ValGeorgiev) [inline packages/image/src/logo/t.js]: Did we test this on web?

Comment (nathggns) [reply packages/image/src/logo/t.js]: This file isn't used on web.

**Pull request merged**